### PR TITLE
Add URLPattern logo

### PR DESCRIPTION
**Pull request by sisidovski**
Adding logo files for the URLPattern API based on the checklist in https://github.com/whatwg/meta/blob/main/NEW-STANDARD.md. The new API is proposed in https://github.com/whatwg/sg/issues/215.

### CODE REVIEW

File: resources.whatwg.org/logo-urlpattern.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+<circle cx="50" cy="50" r="45" fill="#fff" stroke="#3c790a" stroke-width="10"/>
+<path stroke="#3c790a" stroke-width="8" d="M50 22v32M36 28l28 20M64 28 36 48"/>
```
Comment (domenic) [inline resources.whatwg.org/logo-urlpattern.svg:3]:
```suggestion
<path stroke="#3c790a" stroke-width="8" d="M50 34v32M36 40l28 20M64 40L36 60"/>
```

(via https://chat.openai.com/share/7f5fafa5-25cd-4cfd-85e8-dea9a0772d2b )

**Review by domenic — COMMENTED** [review]
Nice work, this is exactly the right approach!

File: resources.whatwg.org/logo-urlpattern.svg (hunk 1 above)
Comment (sisidovski) [reply resources.whatwg.org/logo-urlpattern.svg]: Updated. Thank you AI Domenic.

**Review by domenic — APPROVED** [review]
LGTM. I will delay merging probably until I'm back at work on Monday and we can do all the stuff at once. (That can also give @jeremyroman a chance to take a look if he wants.)

**Pull request merged**